modus-gh-issue-summarizer/README.md

```diff
@@ -0,0 +1,110 @@
+# GitHub Issue Knowledge Base Generator (Modus + Hypermode) 🚀
+
+This project automatically generates a Knowledge Base (KB) article when a GitHub issue is closed. It
+utilizes Hypermode and Modus (a serverless Go framework) to process issue data and generate a
+structured KB article using an LLM. The knowledge base article is then posted as a GitHub
+Discussion.
+
+## 🛠 Features
+
+- Fetches GitHub issue details and comments upon issue closure.
+- Uses LLM (Meta's LLaMa model hosted on Hypermode) to generate a detailed KB article.
+- Posts the generated KB article as a GitHub Discussion.
+- Can be triggered manually or via GitHub Actions.
+
+--
+
+## 🖥️ Running Locally
+
+### 1️⃣ Install Modus
+
+First, install Modus CLI:
+
+```bash
+npm install -g @hypermodeinc/modus
+```
+
+### 2️⃣ Clone the Repository
+
+```bash
+git clone https://github.com/YOUR_USERNAME/YOUR_REPO.git
+cd YOUR_REPO
+```
+
+### 3️⃣ Start the Modus Dev Server
+
+```bash
+modus dev
+```
+
+### 4️⃣ Test the API Locally
+
+Go to: `http://localhost:54321/graphql`
```

Comment:
When I `modus dev` from the new folder in your branch, the handler runs on 8686. Don't see anywhere where this is configured for 54321.